include/EvolutionNet/EvolutionNet.hpp

```diff
@@ -104,8 +113,9 @@ class EvolutionNet {
 };
 
 template <int NumInput, int NumOutput, bool Bias, typename ParamConfig>
-void EvolutionNet<NumInput, NumOutput, Bias, ParamConfig>::initialize(const std::size_t populationSize,
-                                                                      const SeedT rndSeed) {
+void EvolutionNet<NumInput, NumOutput, Bias, ParamConfig>::initialize(
```

Comment:
I would prefer having first argument on the same line. we can adjust the clang format I guess